server.js

```diff
@@ -124,7 +126,7 @@ function render (req, res) {
   })
 }
 
-app.get('*', isProd ? render : (req, res) => {
+app.get('*', (req, res) => {
   readyPromise.then(() => render(req, res))
```

Comment:
It's worth noting that, the then handler is called asyncly, which may have an impact on performance for production mode